Formula/libosinfo.rb

```diff
@@ -20,6 +20,17 @@ class Libosinfo < Formula
   depends_on "glib"
   depends_on "libsoup"
   depends_on "libxml2"
+  depends_on "libarchive"
```

Comment:
It's a nit but I think this should be placed above `libsoup`. I think under `brew audit --strict` it'd complains otherwise, unless I'm imagining that.

Reply:
Ok, will do that right away. I actually thought to put the dependency to `osinfo-db-tools` resource if that is possible.

Reply:
It's not. Sometimes I do things like:
```ruby
depends_on "libarchive" # For osinfo-db-tools
```
so if the resource ever goes _bye-bye_ the depenendecy doesn't _(in an ideal world)_ stick around forever, but that's entirely down to how strongly you feel about it/whether maintainers feel especially strongly about it.

Reply:
Ok, thanks for clarifications!